packages/autocomplete/src/elements/Multiselect.js

```diff
@@ -0,0 +1,449 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { MenuView, Item } from '@zendeskgarden/react-menus';
+import { TextGroup, Label, FauxInput, Input, Hint, Message } from '@zendeskgarden/react-textfields';
+import { Tag, Close } from '@zendeskgarden/react-tags';
+import { Anchor } from '@zendeskgarden/react-buttons';
+import { FieldContainer, KEY_CODES } from '@zendeskgarden/react-selection';
+import AutocompleteContainer from '../containers/AutocompleteContainer';
+
+const DEFAULT_NUM_TAGS = 4;
+const VALIDATION = {
+  SUCCESS: 'success',
+  WARNING: 'warning',
+  ERROR: 'error',
+  NONE: 'none'
+};
+
+const StyledNoItemsMessage = styled.div`
+  margin: 16px;
+  text-align: center;
+`;
+
+const StyledMenuOverflow = styled.div`
+  overflow-y: auto;
+  max-height: ${props => props.maxHeight || 'inherit'};
+`;
+
+const StyledInput = styled(Input)`
+  && {
+    flex-basis: 60px;
```

Comment:
What is this `flex-basis` about? How is the multiselect handling both `small` and default sizing?

Reply:
Addressing the `small` sizing in the comment below

Reply:
Then use `min-width`. I don't think `flex-basis` is doing what you think it is. https://gedd.ski/post/the-difference-between-width-and-flex-basis/

Reply:
There were conflicts with the default width of `100%` that led me down that road. Correcting `width` in combination with `min-width` helped resolve this.